tests/custom-elements/reactions/HTMLButtonElement.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>Custom Elements: CEReactions on Element interface</title>
+<meta name="author" title="Zhang xiaoyu" href="xiaoyux.zhang@intel.com">
+<meta name="assert" content=" autofocus, disabled, formAction, formEnctype, formMethod, formNoValidate, formTarget, name, type, value of HTMLButtonElement interface must have CEReactions">
+<meta name="help" content="https://html.spec.whatwg.org/#htmlbuttonelement">
+<script src="../../resources/testharness.js"></script>
+<script src="../../resources/testharnessreport.js"></script>
+<script src="./custom-elements-helpers.js"></script>
+<script src="./reactions.js"></script>
+</head>
+<body>
+<script>
+
+//IDL attribute reflects the content attribute of the same name inside the parent Node
```

Comment:
Actually I can't understand the comments.

Reply:
Notes are updated.